BackHandler.android.js

```diff
@@ -8,7 +8,7 @@ class BackHandlerAndroid extends React.Component {
 
   constructor(props) {
     super(props);
-    this._didFocusSubscription = props.navigation.addListener('didFocus', payload =>
+    this._didFocusSubscription = props.navigation.addListener('willFocus', payload =>
```

Comment:
thanks! can you please rename this to `_willFocusSubscription`?

Reply:
Oh my mistake. I fix it.